From 7e83d93f8eca2c87126f48f892069b6d3126dbe8 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Mon, 14 Jul 2014 20:25:22 +1000
Subject: [PATCH 214/222] CoreContext: Fix IsThisHost

Wrong parenthesis placement.

(cherry picked from commit 3cb8de1a6feaac3b1320331d3bdfd5a93ae148af)
---
 mythtv/libs/libmythbase/mythcorecontext.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythbase/mythcorecontext.cpp b/mythtv/libs/libmythbase/mythcorecontext.cpp
index c4e3c68..f0260df 100644
--- a/mythtv/libs/libmythbase/mythcorecontext.cpp
+++ b/mythtv/libs/libmythbase/mythcorecontext.cpp
@@ -643,7 +643,7 @@ bool MythCoreContext::IsThisHost(const QString &addr, const QString &host)
     QString thisip  = GetBackendServerIP4(host);
     QString thisip6 = GetBackendServerIP6(host);
 
-    return (!addrfix.isNull() && (addrstr == thisip) || (addrstr == thisip6));
+    return !addrfix.isNull() && ((addrstr == thisip) || (addrstr == thisip6));
 }
 
 bool MythCoreContext::IsFrontendOnly(void)
-- 
1.7.10.2

